app/utils/file-checks/extension-check.js

```diff
@@ -0,0 +1,28 @@
+const allowedExtensions = [
+  'doc',
+  'docx',
+  'xls',
+  'xlsx',
+  'pdf',
+  'jpg',
+  'jpeg',
+  'png',
+  'mpg',
+  'mp4',
+  'wmv',
+  'mov'
+]
+
+function getFileExtension (filename) {
```

Comment:
Same here as my other comment, but any particular reason for using declared functions over arrow functions here on line on 16 and below on line 21?